src/components/MessageList/TypingIndicatorContainer.js

```diff
@@ -0,0 +1,32 @@
+import React, { useContext } from 'react';
+import styled from '@stream-io/styled-components';
+
+import { ChannelContext, ChatContext } from '../../context';
+
+const Container = styled.View`
+  position: absolute;
+  bottom: 0;
+  height: 30px;
+  width: 100%;
+  padding-left: 16px;
+  padding-top: 3px;
+  padding-bottom: 3px;
```

Comment:
oh styled-components... do they not support `padding-vertical` instead of having to declare an individual `bottom` and `top` as the same thing
```suggestion
  bottom: 0;
  height: 30px;
  padding-bottom: 3px;
  padding-left: 16px;
  padding-top: 3px;
  position: absolute;
  width: 100%;
```

Reply:
This is the stuff I had to add to AS I think, let me go grab it, padding-vertical doesn't exist in the linter, I had a list I added I think.

Reply:
ah right, I remember that. may be something we also want to document/bring up to Vish

Reply:
I just added them, there were already 3 in the ignoreProperties array, just had to add the other 11 I know of. Yeah you know my css in js feelings...